### PR TITLE
config: update crda matcher config with `source` and `key`

### PR DESCRIPTION
**Pull request by rafiu007**
This pr is meant to support https://github.com/quay/claircore/pull/423
Signed off: Rafiu Jaman Mollah

### CODE REVIEW

File: Documentation/concepts/matching.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ The matcher API is designed to be called often and will always provide the most
 
 See our [api](../howto/api.md) guide to learn how to view our api specification and work with the Matcher api.
 
+The CRDA remote matcher is responsible for fetching vulnerabilities from Red Hat CRDA. CRDA has the best in industry vulnerability data powered by Snyk.
+This matcher by default  serves 100 requests per minute. Each request can have up to 10 packages for optimum performance.The CRDA matcher comes with a configurable option and we can accommodate a much higher rate of requests. Please fill out [CRDA-Request-Form](https://docs.google.com/forms/d/e/1FAIpQLSfJ1aoOfi0tQ6_o0srEbYL-cSwzo_MyDJQnwcFG9IYVlNXJrA/viewform?usp=sf_link)to avail the configuration details.This can then be used in the clair's config file.
+
```
Comment (crozzy) [inline Documentation/concepts/matching.md:21]: * IMO this deserves a subheader
* Can you add something about why this matcher is different from the normal matching process that refers to Clair's internal DB to surface pre-fetched vulnerabilities? I feel like that could really help someone debugging (i.e. why do I see this vulnerability in my report but I can't find it in my DB?).
* There's a double space in there and a couple of spaces missing after `.`s. 
* Not a part of this PR directly but the form also has some spelling errors: `A breif desciption of use case.`

**Review by crozzy — COMMENTED** [review]
A couple of comments and some nit-picky things

File: Documentation/concepts/matching.md (hunk 1 above)
Comment (rafiu007) [reply Documentation/concepts/matching.md:21]: ACK. I will modify accordingly.

Comment (rafiu007) [reply Documentation/concepts/matching.md:21]: @crozzy Updated it. Decided to create a header  `remote matching` as there is no documentation on that here.

File: Documentation/concepts/matching.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ Matchers by default are also responsible for keeping the database of vulnerabili
...
 
 See our [api](../howto/api.md) guide to learn how to view our api specification and work with the Matcher api.
+# Remote Matching
+A remote matcher behaves similarly to a matcher except that it uses api calls to fetch vulnerabilty data to a provided IndexReport. Remote matchers are usefull when it is not possible to persist data from a given source into the database.
+
+The CRDA remote matcher is responsible for fetching vulnerabilities from Red Hat CRDA. CRDA has the best in industry vulnerability data powered by Snyk. This data cannot be persisted into the db hence the CRDA matcher makes api calls based on the IndexReport to fetch vulnerability data.
+This matcher by default serves 100 requests per minute. Each request can have up to 10 packages for optimum performance. The CRDA matcher comes with a configurable option and we can accommodate a much higher rate of requests. Please fill out [CRDA-Request-Form](https://docs.google.com/forms/d/e/1FAIpQLSfJ1aoOfi0tQ6_o0srEbYL-cSwzo_MyDJQnwcFG9IYVlNXJrA/viewform?usp=sf_link)to avail the configuration details. This can then be used in the clair's config file.
```
Comment (crozzy) [inline Documentation/concepts/matching.md]: Thanks for the updates, I'd just say:
"used in ~the~ ~c~Clair's config file"
and a space after the form link.

Everything else LGTM

Comment (rafiu007) [reply Documentation/concepts/matching.md]: @crozzy Made the requested changes. Thanks for the review comments, if there is nothing else may be you could add approve this pr.

**Review by crozzy — DISMISSED** [review]

**Comment by rafiu007** [comment]
@crozzy Made a very minor change. Also Please approve and retrigger the tests. :)

**Comment by rafiu007** [comment]
@crozzy Squashed the commits and reworded the commit message.This should pass the test.PTAL

**Review by crozzy — DISMISSED** [review]

**Comment by crozzy** [comment]
Checklist for this to be merged:

- [ ] https://github.com/quay/claircore/pull/423 needs to be merged
- [ ] A new claircore tag needs to be generated
- [ ] This PR needs to be updated to included the updated claircore dependency

**Comment by rafiu007** [comment]
@crozzy This should be good to go now PTAL.

**Review by hdonnay — APPROVED** [review]

**Pull request merged**